docs/cmd/tkn_bundle_list.md

```diff
@@ -28,15 +28,16 @@ Authentication:
 	3. Additionally, you can use Basic auth via --remote-username and --remote-password
 
 Caching:
-    By default, bundles will be cached in ~/.tekton/bundles. If you would like to use a different location, set 
-"--cache-dir" and if you would like to skip the cache altogether, set "--no-cache".
+	By default, bundles will be cached in ${XDG_CACHE_HOME}/tekton/bundles. If ${XDG_CACHE_HOME} isn't set then
+	~/.cache/tekton/bundles will be used. If you would like to use a different location, set "--cache-dir" and if
+	you would like to skip the cache altogether, set "--no-cache".
 
 
 ### Options
 
 ```
       --allow-missing-template-keys   If true, ignore any errors in templates when a field or map key is missing in the template. Only applies to golang and jsonpath output formats. (default true)
-      --cache-dir string              A directory to cache Tekton bundles in. (default "~/.tekton/bundles")
+      --cache-dir string              A directory to cache Tekton bundles in. (default "/home/jk/.cache/tekton/bundles")
```

Comment:
the `os.UserXDir()` functions are already expanded. Should I add some logic to un-expand them back to `~/.cache/...`

Reply:
ah 😅 the generate did expand 🤦🏼 yeah we need to fix this somehow 🤔

Reply:
Resolved with:

https://github.com/tektoncd/cli/blob/8c471aac0927659a459065377847e0834d7efbbb/pkg/cli/dirs.go#L48-L49

Might submit a pr to go-homedir to add it in? :shrug: